xlive/H2MOD/Modules/Stats/StatsHandler.cpp

```diff
@@ -592,6 +592,30 @@ void StatsHandler::playerJoinEvent(XUID playerXUID)
 	//the main function.
 	sendRankChange();
 }
+
+static BYTE EmptyByte[2]{ 0, 0 };
```

Comment:
Why don't declare this inside the FixClientPCRRanks() function stack and not static as well? seems to be used only there